src/RunProcessAsTask.Tests/ProcessExTests.cs

```diff
@@ -1,151 +1,156 @@
 ﻿using System;
 using System.Diagnostics;
+using System.IO;
 using System.Linq;
+using System.Reflection;
+using System.Reflection.Internal;
 using System.Threading;
 using System.Threading.Tasks;
+using RunProcessAsTask;
 using Xunit;
 
 namespace RunProcessAsTask.Tests
 {
-    public static class ProcessExTests
-    {
-        public class RunAsyncTests
+        public static class ProcessExTests

```

Comment:
@jamesmanning yeah sure thing. I was using my work VS environment for the last few commits so I'll do that when I get back later.